apps/nextjs-website/src/app/[productSlug]/api/[apiDataSlug]/page.tsx

```diff
@@ -48,7 +48,7 @@ export const generateMetadata = async (
   }
 
   return makeMetadata({
-    title: ApiDataProps?.specURLsName,
+    title: ApiDataProps?.specURLsName + ' | ' + ApiDataProps?.product?.name,
```

Comment:
```suggestion
    title: [ApiDataProps?.specURLsName, ApiDataProps?.product?.name].filter(Boolean).join(' | ')
```
If one of the values is falsey you'll get something like "specUrlsName | " or " | name"